hack/lib/golang.sh

```diff
@@ -558,6 +559,17 @@ kube::golang::build_binaries() {
     local use_go_build
     local -a targets=()
     local arg
+    
+    # Some data files define data that should be generated before compilation.
+    # Lets compile them all in order as necessary.
+    local bindatas=( "./test/e2e/framework/gobindata_util.go" )
```

Comment:
maybe this should be pulled out to a toplevel var at the top of the file like the other lists?